site/management.md

```diff
@@ -275,6 +275,82 @@ rabbitmqctl set_user_tags full_access administrator
 </pre>
 
 
+## <a id="http-api" class="anchor" href="#http-api">Authenticating with OAuth 2</a>
```

Comment:
The anchor here is a copy-paste artifacts and it break on page navigation.

Reply:
Good catch, fixed.